roles/dnsmasq/templates/cifmw-dnsmasq.conf.j2

```diff
@@ -9,4 +9,4 @@ dhcp-leasefile=/var/lib/dnsmasq/cifmw-dnsmasq.leases
 {% endif %}
 
 conf-dir={{ cifmw_dnsmasq_basedir }},*.conf
-dhcp-hostsdir="{{ cifmw_dnsmasq_basedir }}/dhcp-hosts.d"
+dhcp-hostsfile="{{ cifmw_dnsmasq_basedir }}/dhcp-hosts.d"
```

Comment:
From the doc:

> Read DHCP host information from the specified file. If a directory is given,
> then read all the files contained in that directory in alphabetical order. The file
> contains information about one host per line.  The  format of  a line is the same
> as text to the right of '=' in --dhcp-host. The advantage of storing DHCP host
> information in this file is that it can be changed without re-starting dnsmasq: the
> file will be re-read when dnsmasq receives SIGHUP.